### PR TITLE
added code of conduct and updated main DSDay 2024 page

### PR DESCRIPTION
**Pull request by Ciandelle**

### CODE REVIEW

File: src/community/code-of-conduct/index.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+The event organisers reserve the right to ask anyone in violation of these policies to not participate in further activities.
+
+Organisers will reinforce this code throughout event engagements. If you have any concerns, contact any of our organisers immediately. Organisers will be wearing a Design System Day t-shirt.
+
+Alternatively you can [email the Design System Day inbox](design-system-day-enquiries@digital.cabinet-office.gov.uk) and a member of the team will get back to you.
+
+List of organisers:
+
+- Anika Henke
```
Comment (kellylee-gds) [inline src/community/code-of-conduct/index.md:31]: Anika doesn't have a volunteer role at the event so can be removed.

File: src/community/code-of-conduct/index.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+- Bee Butler
+- Calvin Lau
+- Chris Ballantine-Thomas
+- Imran Hussain
+- Izabela Podralska
+- Kam Nijjar
+- Katrina Birch
+- Kelly Lee
+- Kim ‘beeps’ Grey
+- Mia Allers
```
Comment (kellylee-gds) [inline src/community/code-of-conduct/index.md:41]: Mia doesn't have a volunteer role at the event so can be removed.

File: src/community/code-of-conduct/index.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+- Nora Brodian
+- Owen Jones
+- Patrick Cartlidge
+- Romaric Pascal
+- Tim Paul
```
Comment (kellylee-gds) [inline src/community/code-of-conduct/index.md:46]: Not sure about Tim yet either

**Review by kellylee-gds — CHANGES_REQUESTED** [review]
Looks good! I'd just remove the names I've mentioned then we're good to go

**Review by Ciandelle — COMMENTED** [review]
changes made as requested

**Comment by kellylee-gds** [comment]
Thanks, and approved!

**Review by kellylee-gds — APPROVED** [review]

**Pull request merged**